src/twisted/newsfragments/9660.bugfix

```diff
@@ -0,0 +1 @@
+twisted.internet._glibbase.PortableGlibReactorBase.simulate no longer raises TypeError when there are no delayed called. This was a regression introduced with the migration to Python3 in which the builtin `min` function no longer accepts `None` as an argument.
```

Comment:
The relevant public names are `twisted.internet.gireactor.PortableGIReactor.simulate` and `twisted.internet.gtk2reactor.PortableGtkReactor.simulate`; in general we should avoid news fragments that mention private names (those which have a single underscore directly following a dot)

Reply:
so....should I change something or you guys gonna figure this one out?

Reply:
We can go ahead and merge this as-is, my comment was more for @adiroiban than you :)

Reply:
(Nothing wrong with fixing up the changelog later.)